badgr-fullstack-ubuntu/README.md

```diff
@@ -0,0 +1,21 @@
+# Deploy Badgr-Server FullStack on Ubuntu
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fsatyarapelly%2Fazure-quickstart-templates%2Fmaster%2Fbadgr-fullstack-ubuntu%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2Fsatyarapelly%2Fazure-quickstart-templates%2Fmaster%2Fbadgr-fullstack-ubuntu%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template deploys the Badgr Server web application (concentricsky) on Ubuntu. 
+
+Connect to the virtual machine with SSH: `ssh {adminUsername}@{dnsNameForPublicIP}.{region}.cloudapp.azure.com`. Installation log can be found under */var/log/azure*.
```

Comment:
this uri will vary by cloud - remove this and use an output to give the user the ssh cmd